blueocean-dashboard/src/main/js/components/Branches.jsx

```diff
@@ -12,6 +12,29 @@ import { CellRow, CellLink } from './CellLink';
 
 import { buildRunDetailsUrl } from '../util/UrlUtils';
 
+function noRun(branch, openRunDetails, t, store) {
+    return (<tr>
+                <td></td>
+                <td></td>
+                <td>{decodeURIComponent(branch.name)}</td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td className="actions">
+                    <RunButton
+                      className="icon-button"
+                      runnable={branch}
+                      onNavigation={openRunDetails}
+                    />
+                    <Extensions.Renderer
+                      extensionPoint="jenkins.pipeline.branches.list.action"
+                      pipeline={branch }
+                      store={store}
```

Comment:
I noticed that here you are passing the `store` and `pipeline` to `extensionPoint="jenkins.pipeline.branches.list.action"` but in the PR one not. Why? They are the same ExtensionPoints I expect they would need the same props.